mesheryctl/cmd/status.go

```diff
@@ -28,12 +27,12 @@ var statusCmd = &cobra.Command{
 	Short: "Check Meshery status",
 	Long:  `Check status of Meshery and Meshery adapters.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Meshery containers status . . .\n")
+		log.Info("Meshery containers status . . .\n")
 		out, err := exec.Command("docker-compose", "-f", dockerComposeFile, "ps").Output()
 		if err != nil {
 			log.Fatal("[ERROR] Please, install docker-compose. The error message: \n", err)
 		}
-		fmt.Printf("%s\n", out)
+		log.Printf("%s\n", out)
```

Comment:
change Printf to Infof? and we probably don't want \n

Logrus does a good job with printing each log in separate lines. Can you please also get rid of all the "\n" in print statements in mesheryctl?

Reply:
Yeah thanks for the feedback, and consider it done.
Thanks @girishranganathan.